api/hypershift/v1beta1/hosted_controlplane.go

```diff
@@ -200,6 +200,18 @@ type HostedControlPlaneSpec struct {
 	// +kubebuilder:validation:MaxProperties=20
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
+
+	// DisabledCapabilities when specified, sets the cluster version BaselineCapabilitySet to None
+	// and sets all AdditionalEnabledCapabilities BUT the ones supplied in DisabledCapabilities.
+	// This effectively disables that capability on the hosted cluster.
+	//
+	// When this is not supplied, the cluster will use the DefaultCapabilitySet defined for the respective
+	// OpenShift version.
+	//
+	// +listType=atomic
+	// +optional
+	// +openshift:enable:FeatureGate=DisableClusterCapabilities
+	DisabledCapabilities []OptionalCapability `json:"disabledCapabilities,omitempty"`
```

Comment:
you'll need to add the featuregate to the files within v1beta1/featuregates/* then run make update to generate the CRDs

Reply:
done.